notifications/NotificationFCM.cpp

```diff
@@ -95,18 +95,33 @@ bool CNotificationFCM::SendMessageImplementation(
 {
 	//send message to FCM
 
-	std::string sMidx;
+	uint64_t AltIdx = Idx;
```

Comment:
No need to declare a new uint64, as Idx parameter is already a FROM device idx. All the logic is here already to handle the sending divce, no need to get it from extraData here.
Maybe we need to change the call to the SendMessageImplementation, so de from device is removed from extraData and put in as first argument?

Reply:
In most cases that is indeed correct.

Unfortunately, when a user uses the dzVents `notify()` function, no FROM device is passed (as there is no parameter in dzVents to pass this).
By _misusing_ the extraData parameter to pass a device Idx (see modification in Eventsystem.cpp), this omission in circumvented.
Hence we need an AltIdx (as the Idx is a `const` (which is ought to be anyway when done properly)).

Off course, a better solution would be to _fix_ the dzVents notify function. (@gizmocuz , any thoughts?)